src/main/java/com/debatetimer/dto/member/JwtTokenResponse.java

```diff
@@ -0,0 +1,4 @@
+package com.debatetimer.dto.member;
+
+public record JwtTokenResponse(String accessToken, String refreshToken) {
+}
```

Comment:
[사소한 제안]
컨벤션에 따라 한줄 개행 부탁드립니다 😄 
```suggestion

}
```

Reply:
수정했습니다